packages/@aws-cdk/aws-ecs/test/ec2/ec2-service.test.ts

```diff
@@ -3280,24 +3307,61 @@ describe('ec2 service', () => {
   });
 
   describe('When import an EC2 Service', () => {
-    test('with serviceArn', () => {
+    test('with old serviceArn', () => {
```

Comment:
Testing the new vs old service ARN format should be a part of https://github.com/aws/aws-cdk/pull/18140. It is out of scope for this one.